larpandoracontent/LArObjects/LArSupportVectorMachine.h

```diff
@@ -179,6 +188,10 @@ class SupportVectorMachine
     typedef std::map<KernelType, KernelFunction> KernelMap;
 
     bool              m_isInitialized;       ///< Whether this svm has been initialized
+    
+    bool              m_enableProbability;   ///< Whether to enable probability calculations
+    double            m_probAParameter;      ///< The 'A' parameter for mapping to probailities
```

Comment:
Typo: probailities

Any insight as to exactly what this parameter represents (and 'B' too)?